scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -2,82 +2,12 @@
 
 namespace Microsoft.DotNet.Cli.Build
 {
-    public class BuildVersion
+    public class BuildVersion : Version
     {
-        public int Major { get; set; }
-        public int Minor { get; set; }
-        public int Patch { get; set; }
-        public int CommitCount { get; set; }
-        public string CommitCountString => CommitCount.ToString("000000");
-        public string ReleaseSuffix { get; set; }
-
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
-        public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
+        public string VersionSuffix => $"{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
```

Comment:
@Sridhar-MS are you going to change this back? I thought that's what we talked about